src/Navigation/Xamarin.Forms/Navigation.Xamarin.Forms.Interfaces/Navigation.Xamarin.Forms.Interfaces.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <AssemblyName>CodeMonkeys.Navigation.Xamarin.Forms.Interfaces</AssemblyName>
+    <RootNamespace>CodeMonkeys.Navigation.Xamarin.Forms.Interfaces</RootNamespace>
+    <PackageId>CodeMonkeys.Navigation.Xamarin.Forms.Interfaces</PackageId>
+    <Version>2.0.0</Version>
```

Comment:
2.0.0? Or should we increase it to the same version when releasing as the other packages?